docs/user-guide/helm.md

```diff
@@ -1,56 +1,76 @@
-# Installing Ambassador with Helm
+# Installing the Ambassador Edge Stack with Helm
 
-[Helm](https://helm.sh) is a package manager for Kubernetes that automates the release and management of software on Kubernetes. The Ambassador Edge Stack can be installed via a Helm chart with a few simple steps, depending on if you are deploying for the first time, or upgrading from an existing installation.
+[Helm](https://helm.sh) is a package manager for Kubernetes that automates the release and management of software on Kubernetes. The Ambassador Edge Stack can be installed via a Helm chart with a few simple steps, depending on if you are deploying for the first time, upgrading the Ambassador Edge Stack from an existing installation, or migrating from the Ambassador API Gateway.
 
-## Prerequisites
+## Before You Begin
 
-The Ambassador Edge Stack Helm chart is hosted by Datawire and published at `https://www.getambassador.io`. Start by adding this repo to your helm client with:
+The Ambassador Edge Stack Helm chart is hosted by Datawire and published at `https://www.getambassador.io`.
+
+Start by adding this repo to your helm client with the following command:
 
 ```bash
 helm repo add datawire https://www.getambassador.io
 ```
 
-## First Time Installation
+## Install with Helm
+
+When you run the Helm chart, it installs the Ambassador Edge Stack. You can
+deploy it with either version of the tool. 
+
+1. If you are installing the Ambassador Edge Stack **for the first time on your cluster**, create the `ambassador` namespace for the Ambassador Edge Stack:
 
-If you are installing the Ambassador Edge Stack for the first time on your host, complete the following directions:
+   ```
+   kubectl create namespace ambassador
+   ```
 
-1. Create the `ambassador` namespace for the Ambassador Edge Stack:
+2. **Helm 3 users:** Install the Ambassador Edge Stack Chart with the following command:
 
-    ```
-    kubectl create namespace ambassador
-    ```
+   ```
+   helm install ambassador --namespace ambassador datawire/ambassador
+   ```
 
-2. If you are using Helm 3, install the Ambassador Edge Stack Chart with the following command:
+3. **Helm 2 users**: Install the Ambassador Edge Stack Chart with the following command:
```

Comment:
I think it's confusing there's a step 3 and a step 2 when it's either/or.

Reply:
fixed